src/components/KeyboardShortcutsModal.js

```diff
@@ -41,7 +41,7 @@ const openShortcutModalConfig = CONST.KEYBOARD_SHORTCUTS.SHORTCUT_MODAL;
 function KeyboardShortcutsModal({isShortcutsModalOpen = false, isSmallScreenWidth, translate}) {
     const subscribedOpenModalShortcuts = useRef([]);
     const modalType = isSmallScreenWidth ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.CENTERED_UNSWIPEABLE;
-    const shortcuts = KeyboardShortcut.getDocumentedShortcuts();
+    const [shortcuts, setShortcurts] = useState([]);
```

Comment:
@Charan-hs May I know what the reason for doing this? Are there any benefits or drawbacks from the previous code?

Reply:
Thanks @Charan-hs! Make sense to me. Just to make I didn't slip out any of the purpose changes.